### DISCUSSION TITLE
PlaceholderAPI

### DISCUSSION THREAD

**Discussion by blueawp** [GriefPrevention/GriefPrevention#926] [Q&A] [answered]
when i updated to the latest placeholderAPI  "2.10.8* the placeholder {placeholderapi_griefprevention_currentclaim_ownername_color}
stop working

**Comment by Jikoo** [comment, marked as answer]
Duplicate of #923

**Comment by Jikoo** [comment]
> GP does not provide a PAPI hook, PAPI offers a GP hook. Try re-downloading the hook via their system, and if you still encounter the issue contact PAPI.
